### DISCUSSION TITLE
你家貓咪其實都在偷偷觀察你喔

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1412] [General]
大家會不會覺得你家的貓都對你愛理不理的，每次跟貓互動都不理我
我就在想，他真的有在 care 你是死是活嗎？​
之前在網路上看到有​日本研究團隊找來數十隻貓咪做實驗​，他們分別在不同房間中播放主人的聲音​，再用攝影機偷偷觀察每隻貓的反應​，結果發現儘管貓咪看上去都在放空​，可當主人的聲音從原本的A房​突然跳到幾十公尺之外的B房時​，這些貓大多會猛然抬頭，這顯示祂們其實都已經掌握到主人的位置​，科學家因此推測，貓能根據環境中的聲音​判斷出你是誰、你在哪、甚至是你在做什麼。
結論：貓咪只是比較傲嬌而已，他還是很愛你的....吧？

**Comment by Brownnnn78** [comment]
真的超級愛貓咪的每天都要在挨居攝取大量貓咪影片，尤其是那種腿短的小貓咪，可惜還沒有能力養貓。總之有貓就給讚。

**Comment by Jamesmoneya** [comment]
貓咪真的很極端，要可愛的時候可以很可愛，但是當他傲嬌起來就不會管你是不是他的主人，他就只會對你一直嘶吼然後抓你，如果沒有拍下貓咪可愛的影片，我真的會放棄養貓

**Comment by alicia0725** [comment]
真的有貓咪都在偷偷觀察的感覺，有時候突然移動想說貓咪沒在看結果就又突然出現在我旁邊了

**Comment by ALAN30359704** [comment]
我對我們家的貓咪的確是又愛又恨我們家的貓咪在吃東西的時候最可愛，但是當他想睡覺，但是你不上樓睡覺時候他就會不斷地咬你甚至把我們家的人都咬到流血真的非常的可惡但是看到他圓滾滾的肚子跟圓圓的頭又不知不覺地笑了！

**Comment by alstinlin** [comment]
難怪有時候想從貓的背後偷偷伸手去摸都會被發現，而且有時候雖然沒有發出聲音，貓好像還是能感覺到氣流的變化，總是會在我摸到的前一秒張開眼睛，超級可愛，真希望以後有機會也可以養一隻貓

**Comment by ashleyy-huang** [comment]
雖然我沒有養貓，但看完大家的留言覺得貓咪好神奇，時不時也會在社交軟體看到有人放一些與貓咪互動的影片或是照片，我也希望有朝一日可以與貓咪憶起體驗生活，好好了解一下這個令人又愛又恨的可愛動物。

**Comment by zephyrxin01** [comment]
我也沒有養過貓貓，但是常常看貓咪的療癒影片。感覺貓貓是傲嬌吧~而且有了數據支持感覺這個論點應該不是貓奴們的臆想吧。

**Comment by Brownnnn78** [comment]
本來覺得網路會追蹤我看的影片，哪些內容停留的時間比較久，這樣的行為很恐怖，但是我現在超喜歡instagram 關注我看的影片，他會推薦更多更多貓貓的影片給我！

**Comment by zhiron544** [comment]
雖然我沒有養貓，但我知道貓咪是種傲嬌的生物。他有著可以360度去聽見周圍所有聲音跟立刻判斷位置的肌肉骨骼，但它也只對罐頭跟食物有反應。
